### DISCUSSION TITLE
Carousel autoplay  transition without stopping in the middle with smooth transition

### DISCUSSION THREAD

**Discussion by Arunjenson-ss** [davidjerleke/embla-carousel#548] [Help] [answered]
@davidjerleke i need to implement an autoplay carousel where i don't need to stop , it needs to continuously move with the smooth transition

**Comment by davidjerleke** [comment, marked as answer]
Hi @Arunjenson-ss,

Thank you for your question. Do you mean a **continuously running carousel** like a ticker? If yes, I shared a **manual implementation** that works for older versions of Embla here: 

- #114

I haven't had the time to adapt it to the latest version. It's already on my to do list to **build a plugin** called `embla-carousel-auto-scroll` which solves this. It's mentioned here:

- #284

But my spare time is very limited so you can either build the plugin yourself, or consider supporting this project by sponsoring it or find people who can. Thank you for understanding.

Best,
David

**Comment by NickWoodward** [comment]
Just posting this again on the off chance it's helpful (rather than hacky): Setting the duration to something long and the auto play delay to zero seems to work:

```  
const autoplayOptions = {  delay:0,  rootNode: (emblaRoot) => emblaRoot.parentElement}
const options = { loop: true, duration: 4000 }

// stuff

const [emblaRef, emblaApi] = useEmblaCarousel(options, [Autoplay(autoplayOptions)])

  ```

**Comment by davidjerleke** [comment]
Closing because planned in:

- #284
